controllers/storageautoscaler/verify_test.go

```diff
@@ -0,0 +1,30 @@
+package storageautoscaler
+
+import (
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestFailIfTimeOut(t *testing.T) {
+	timeoutSeconds := 100
+	startTime := metav1.Time{
+		Time: metav1.Now().Time,
+	}
+
+	fail := failIfTimeOut(timeoutSeconds, startTime)
+	if fail {
+		t.Errorf("Expected false, got %v", fail)
+	}
+
+	timeoutSeconds = 2
+	startTime = metav1.Time{
+		Time: metav1.Now().Add(-2 * time.Second),
+	}
+
+	fail = failIfTimeOut(timeoutSeconds, startTime)
+	if !fail {
```

Comment:
Instead of checking the bool and calling `t.Errorf()`, just call something like `assert.True(...)`